cranelift-codegen/meta/src/shared/legalize.rs

```diff
@@ -260,11 +263,13 @@ pub(crate) fn define(insts: &InstructionGroup, imm: &Immediates) -> TransformGro
     );
 
     narrow.legalize(
-        def!(brnz.I128(x, ebb, vararg)),
+        def!(brnz.I128(x, ebb1, vararg)),
         vec![
             def!((xl, xh) = isplit(x)),
-            def!(brnz(xl, ebb, vararg)),
-            def!(brnz(xh, ebb, vararg)),
+            def!(brnz(xl, ebb1, vararg)),
+            def!(jump(ebb2, Literal::empty_vararg(&imm.vararg))),
+            ebb!(ebb2),
```

Comment:
```suggestion

            ebb!(ebb2),
```

Reply:
Not sure if `cargo fmt` will like this change.

Reply:
I think cargo fmt doesn't run in macro's code (probably because the formatting could be part of the macro pattern matching), but not sure.

Reply:
After testing, `cargo fmt` does re-indent and remove the empty line.

Reply:
> probably because the formatting could be part of the macro pattern matching

No it can't. I think it doesn't fmt macro invocations because rustfmt doesn't know if a certain part of a macro invocation is an expr, item, pattern, ... However it seems that blank lines are removed.